### DISCUSSION TITLE
Set styles on FloatingPortal node

### DISCUSSION THREAD

**Discussion by codebex** [floating-ui/floating-ui#3053] [Q&A] [answered]
I have an issue with the` <FloatingPortal />` component where I need to adjust the `z-index` of the portal node. This is necessary because other elements on the page (which I cannot modify) have `z-index` values resulting in elements appearing over the tooltip.

Is there a way to set CSS properties on the portal or any alternative solution?

A similar question was posted last year, but the answer was to render the portal conditionally, and I am already doing this.

**Comment by atomiks** [comment, marked as answer]
You can just set the z-index to the maximum (`2 ** 31 - 1`) on the floating element itself
